src/gui/MainWindow.cpp

```diff
@@ -519,6 +519,11 @@ void MainWindow::updateWindowTitle()
         windowTitle = BaseWindowTitle;
     } else {
         windowTitle = QString("%1 - %2").arg(customWindowTitlePart, BaseWindowTitle);
+        if (customWindowTitlePart.right(1) == "*") {
```

Comment:
@droidmonkey Just updated to check the status directly from dbStruct instead of the window title